target/linux/ath79/dts/qca9558_mikrotik_routerboard-922uags-5hpacd.dts

```diff
@@ -0,0 +1,187 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	compatible = "mikrotik,routerboard-922uags-5hpacd", "qca,qca9558";
+	model = "MikroTik RouterBOARD 922UAGS-5HPacD";
+
+	aliases {
+		label-mac-device = &eth0;
+		led-boot = &led_user;
+		led-failsafe = &led_user;
+		led-upgrade = &led_user;
+		serial0 = &uart;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		led_user: user {
+			label = "mikrotik:green:user";
```

Comment:
ar71xx used rb:green:user, so you might want to add a migration to led-migrations. Maybe make it generic already, something like replacing "^rb:" by "mikrotik".

Reply:
I added this, which should work for any future MikroTik board to be added:
```
case "$board" in
mikrotik,*)
	migrate_leds "^rb:=mikrotik:"
	;;
esac
```

Reply:
Though I do not really like wildcards in cases (it's terrible if you grep for a device name), I think in this case it might be a good idea.